Sources/DittoSyncStatusHelper/DittoSubscriptionsStatusHelper.swift

```diff
@@ -59,6 +59,22 @@ public class DittoSubscriptionsStatusHelper {
         }
     }
 
+    /**
+     Creates a new` DittoSyncStatusHelper` for all of the currently active subscriptions on this Ditto instance *at the time this is created*. It will not update if those subscriptions change
```

Comment:
nit: Looks like there's a space after the first backtick '`' before DittoSyncStatusHelper

Reply:
Fixed